code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -442,7 +442,7 @@
 	desc = "Causes kinetic accelerator shots to slightly heal the firer on striking a living target."
 	icon_state = "modkit_crystal"
 	modifier = 2.5 //Not a very effective method of healing.
-	cost = 20
+	cost = 0
```

Comment:
Can you put multiple of these in 1 gun?

Reply:
Yes you can. Acquiring 2 or more requires quite a bit of luck though, considering they're a low RNG drop from tendril chests. The mod is also quite weak, even if stacked. It restores 2.5 damage per crystal per hit.